File: DingRTC/Server/csharp/AppToken/Util/EncodeUtils.cs
namespace AppToken.Util
{
    public static class EncodeUtils
    {
        public static String Base64Encode(byte[] data)
        {
            return Convert.ToBase64String(data);
        }

        public static byte[] Base64Decode(string data){
            return Convert.FromBase64String(data);
        }
    }
}

File: Server/java/src/main/java/com/company/apptoken/util/EncodeUtils.java
package com.company.apptoken.util;

import org.apache.commons.codec.binary.Base64;

public class EncodeUtils {
    public static String base64Encode(byte[] data) {
        final byte[] encodedBytes = Base64.encodeBase64(data);
        return new String(encodedBytes);
    }

    public static byte[] base64Decode(String data) {
        return Base64.decodeBase64(data.getBytes());
    }
}
